Zend/zend_exceptions.c

```diff
@@ -753,11 +753,6 @@ void zend_register_default_exception(void) /* {{{ */
 	zend_ce_error_exception = register_class_ErrorException(zend_ce_exception);
 	zend_ce_error_exception->create_object = zend_error_exception_new;
 
-	/* Declared manually because it uses constant E_ERROR. */
```

Comment:
Another slight additional benefit of adding constants to stubs is that some properties with constant default values could be autogenerated.